src/datasets/load.py

```diff
@@ -1159,7 +1159,10 @@ def dataset_module_factory(
                     elif "401" in str(e):
                         msg = f"Dataset '{path}' doesn't exist on the Hub"
                         msg = msg + f" at revision '{revision}'" if revision else msg
-                        raise FileNotFoundError(msg + ". If the repo is private, make sure you are authenticated.")
+                        raise FileNotFoundError(
+                            msg
+                            + ". If the repo is private, make sure you are authenticated with ``use_auth_token=True`` after logging in with ``huggingface-cli login```."
```

Comment:
Two nits:
- I would not use double backticks here: this is written to the shell (no Markdown/reStructuredText rendering is going here)
- Anyway, there is a wrong triple backtick at the end.